builder/config.go

```diff
@@ -19,6 +19,8 @@ type Config struct {
 	RemoteRelayEndpoint           string   `toml:",omitempty"`
 	SecondaryRemoteRelayEndpoints []string `toml:",omitempty"`
 	ValidationBlocklist           string   `toml:",omitempty"`
+	RelayConfigFile               string   `toml:",omitempty"`
```

Comment:
I understand the appeal of adding relay config to a separate file, but I feel that this complicates config setup.

Right now, we have two ways of setting up builder config:
1. toml
2. cli params
you have to look at two places to know the builder config. (one exception is blacklist.json, but this one more like "data" thing than basic config)

Relay config file will be part of config separated and moved to json.

Why don't we map relay config into config way that we already have?

1. in toml relay config will be in place
2. in cli relay config can be added as an optional modification to the relay url that will look like:
`--builder.remote_relay_endpoint='URL;ssz;gzip'`
and the same thing with secondary endpoint, but they will be comma separated like today.


cc @metachris @Wazzymandias

Reply:
i think that would make sense, and it's probably better than using multiple config files